packages/landing/static/index.css

```diff
@@ -13,7 +13,10 @@ button {
 
 #map {
     width: 100%;
-    height: calc(100% - 58px);
+    left: 0;
```

Comment:
the calc was here for browser compatilibty, does this work on everything, ios? safari? ie11? firefox?

Reply:
the `- 58px` was wrong for my browser. It does work for me in firefox, google and webkit2 but will switch to a flexbox to ensure it is more progressive.

Reply:
Now uses `flexbox`. There were a bunch of errors in the `index.html` so I fixed those up as well.

Reply:
We might need to do a bunch of testing across browsers on this, I wonder if there is a easy way to automate it